### ISSUE TITLE
How to run testing on file protocol?

### ISSUE THREAD

**Issue by SLyHuy** [B2F/Succss#2]
Hi,

I try to test website on local machine by using:

```
url:'../index.html',
```

But SUCCSS can not work.
I know BackstopJS and PhantomCSS can do.

Thanks,
Huy Ly.

**Comment by B2F** [comment]
It might be possible to update the source at what is currently https://github.com/B2F/Succss/blob/v1.7.2/succss.js#L183

We would need another case detecting relative path to local files and formatting them without 'http://', I think it should work. Atm, you can try editing your succss.js file locally.

Unfortunately I don't plan to add features to Succss v1 as of now, only major blocking bugfixes.
I plan to release a new version of the tool based on WebdriverIO instead of CasperJS. I will be post announcement of the next major version on the github project home.

So to sum up, for this to be working out of the box, you need a webserver running locally at least.

Thanks for your interested in this project SLyHuy !

**Comment by B2F** [comment]
Feel free to fork tho !
